desktop/apps/artwork/components/partner_stub/index.coffee

```diff
@@ -1,11 +1,16 @@
+InquireViaPhoneModalView = require './components/inquire_via_phone_modal.coffee'
+
 module.exports = ($el) ->
   $el
-    .find '.js-artwork-partner-stub-phone-toggle'
+    .find '.js-artwork-partner-stub-inquire-via-phone'
     .click (e) ->
       e.preventDefault()
-
-      $(this).hide()
-
-      $el
-        .find '.js-artwork-partner-stub-phone-toggleable'
-        .show()
+      @modal = new InquireViaPhoneModalView
+        width: '500px'
+        title: 'Call gallery'
+        copy:
+          login: 'Log in to Artsy to call gallery'
+          signup: 'Create an Artsy account to call gallery'
+          register: 'Create an Artsy account to call gallery'
+        userData:
+          partner: JSON.parse(e.toElement.dataset.partner)
```

Comment:
I wasn't actually sure if this is good... Also, I don't think this is the intended use of `userData` since `partner` is not really user data. I just used it so I have access to partner data. Another room for improvement.